pennylane/ops/cv.py

```diff
@@ -709,16 +709,16 @@ class GaussianState(CVOperation):
 
     **Details:**
 
-    * Number of wires: None (applied to the entire subsystem)
+    * Number of wires: Any
     * Number of parameters: 1
-    * Gradient recipe: None (uses finite difference)
+    * Gradient recipe: None
 
     Args:
         r (array): a length :math:`2N` vector of means, of the
             form :math:`(\x_0,\dots,\x_{N-1},\p_0,\dots,\p_{N-1})`
         V (array): the :math:`2N\times 2N` (real and positive definite) covariance matrix
     """
-    num_wires = 0
+    num_wires = Any
     num_params = 2
     par_domain = "A"
     grad_method = "F"
```

Comment:
the grad_method should be `None` here, right? The state preparation should have no gradient recipe and no gradient method.

Reply:
Yep, this should be updated. With regards to the `grad_method` attribute, I decided to leave it for this PR, as it is out of scope (this PR is only concerned with fixing `num_wires`.

At some point we should go through and clarify all the gradient recipes.

Reply:
ok :)  I will make a separate PR for that